include/tim/vx/ops/depth2space.h

```diff
@@ -48,8 +48,8 @@ namespace ops {
 class DepthToSpace : public BuiltinOp {
  public:
   enum depth2space_mode {
-    CRD_mode = 0,
-    DCR_mode = 1,
+    DCR_mode = 0,
+    CRD_mode = 1,
```

Comment:
What's the difference? I didn't see anyone use "0" or "1" directly.

Reply:
@sunshinemyson 
The value of DCR_ mode and CRD_mode is contrary to the definition of ovxlib.
https://github.com/VeriSilicon/TIM-VX/blob/922d10696dbd2475b1d056ef574049e92401cc96/src/tim/vx/internal/include/vsi_nn_types.h
<img width="338" alt="Screen Shot 2023-01-13 at 2 56 55 PM" src="https://user-images.githubusercontent.com/56160181/212256887-25c09350-da79-4c6c-8064-e041033d7137.png">

Reply:
Got it. Thanks